builds/jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -447,6 +501,10 @@ def servelessDeploy(env, envBucketKey){
   return outputLog
 }
 
+def getApiIDName(accountDetails) {
```

Comment:
I dont think this method needed, perhaps it is confusing. Use the awsAPIGatewayModule.getApiGatewayCore(accountDetails) directly for getting api core services details. Please do in other workflows too.

Reply:
removed as advised